Source/Core/DiscIO/NANDContentLoader.cpp

```diff
@@ -398,16 +398,16 @@ void CNANDContentLoader::RemoveTitle() const
   }
 }
 
-cUIDsys::cUIDsys()
+cUIDsys::cUIDsys(Common::FromWhichRoot root)
 {
-  UpdateLocation();
+  Update(root);
 }
 
-void cUIDsys::UpdateLocation()
+void cUIDsys::Update(Common::FromWhichRoot root)
```

Comment:
Is there any point in having this as a separate function anymore? Maybe we should just force calling code to construct a new object if they for some reason want to change root.